NDcPP_v2_2e.adoc

```diff
@@ -1965,6 +1971,34 @@ _If the identifier is not a FQDN, then the TSS shall describe how the identifier
 
 _The client identifier may be in the Subject field or the Subject Alternative Name extension of the certificate. The expected identifier may either be configured, may be compared to the FQDN, IP address, username, or email address used by the client, or may be passed to a directory server for comparison._
 
+*FCS_TLSS_EXT.2.4* The TSF shall [selection:
+
+* _present a TLS1.1 Certificate Request message containing the following ClientCertificateTypes: [selection:_
+** _rsa_sign,_
+** _ecdsa_sign,_
+** _] and no other ClientCertificateTypes;_
+* _present a [selection: TLSv1.2, TLSv1.3] Certificate Request message containing the following algorithms: [selection:_
+** _rsa_pkcs1 with sha256(0x0401),_
+** _rsa_pkcs1 with sha384(0x0501),_
+** _rsa_pkcs1 with sha512(0x0601),_
+** _ecdsa_secp256r1 with sha256(0x0403),_
+** _ecdsa_secp384r1 with sha384(0x0503),_
+** _ecdsa_secp521r1 with sha512(0x0603),_
+** _rsa_pss_rsae with sha256(0x0804),_
+** _rsa_pss_rsae with sha384(0x0805),_
+** _rsa_pss_rsae with sha512(0x0806),_
+** _rsa_pss_pss with sha256(0x0809),_
+** _rsa_pss_pss with sha384(0x080a),_
+** _rsa_pss_pss with sha512(0x080b)_
+** _] and no other algorithms;_ +
+_]._
+
+*_Application Note {counter:appnote_count}_*
+
+_The selected ClientCertificateTypes and algorithms must be consistent with FCS_COP.1/SigGen and FCS_COP.1/Hash._ +
+_The TLS 1.2 Certificate Request message includes the algorithms as a list of  SignatureAndHashAlgorithms in the supported signature algorithms list. The signature algorithm is also specified in the certificate_types list._ +
+_The TLS 1.3 Certificate Request message includes the algorithms as a list of SignatureSchemes in the signature_algorithms extension._
+
```

Comment:
Please check there is a line break at the end of each sentence of the Application Note.

Reply:
@plughy2 When I view this from my temporary branch, tls-110521-kk. I don't see an issue with formatting. Maybe we can discuss this on the call today.

Reply:
@plughy2 I made edits that should have fixed this.